react-frontend/src/components/Home/cardlist.js

```diff
@@ -58,7 +59,7 @@ function CardList() {
                 title={'Digital Policy'}
                 img={imgLinks[2]}
                 description={descriptions[2]}
-                linkText={linkTexts[2]}
+                // linkText={linkTexts[2]}
```

Comment:
whats with the commenting out? if not needed remove :):

Reply:
Will need later!